{
    "config": {
        "error": {
            "already_configured": "O dispositivo j\u00e1 est\u00e1 configurado",
            "cannot_connect": "Falha na liga\u00e7\u00e3o"
        },
        "step": {
            "init": {
                "data": {
                    "id": "",
                    "name": "Nome"
                }
            }
        }
    }
}